app/src/test/java/net/skyscanner/backpack/BpkSnapshotTest.kt

```diff
@@ -40,7 +40,7 @@ import org.robolectric.annotation.GraphicsMode
 
 @RunWith(BpkTestRunner::class)
 @GraphicsMode(GraphicsMode.Mode.NATIVE)
-@Config(qualifiers = RobolectricDeviceQualifiers.SmallPhone, sdk = [28])
+@Config(qualifiers = RobolectricDeviceQualifiers.SmallPhone, sdk = [33])
```

Comment:
we also had to bump the sdk we run the tests on for these to work. while we generally tried to run snapshot tests on the lowest sdk this is much less critical these days with compose + app compat libraries